pftp/data_handler.go

```diff
@@ -351,6 +356,20 @@ func (d *dataHandler) clientListenOrDial() error {
 		d.clientConn.dataConn = tcpConn
 	}
 
+	if d.needTLSForTransfer {
+		if d.tlsDataSet.forClient.getTLSConfig() == nil {
+			return errors.New("cannot get TLS config for data transfer. abort data transfer")
+		}
+
+		tlsConn := tls.Server(d.clientConn.dataConn, d.tlsDataSet.forClient.getTLSConfig())
+		if err := tlsConn.Handshake(); err != nil {
+			d.log.err("TLS client data connection handshake got error: %v", err)
```

Comment:
why don't you return err?
I want to hear your intentions.

Reply:
Oh, I'm sorry. It was just miss.
It's vestiges of development... 🙏 

Fix at https://github.com/pyama86/pftp/pull/38/commits/a5f2f3cdf6c17712861647ee6f351238e57e31fb